lib/Text/Xslate.pm

```diff
@@ -623,8 +623,8 @@ sub _compiler {
     my $compiler = $self->{compiler};
 
     if(!ref $compiler){
-        require Mouse;
-        Mouse::load_class($compiler);
+        require Class::Load;
```

Comment:
I would add Class::Load::XS to the list of optional dependencies -- e.g. via Dist::Zilla::Plugin::DynamicPrereqs `require('Class::Load::XS') if not want_pp() and can_xs();` (or the equivalent unrolled code) in Makefile.PL.